packages/urban-bot/src/components/Router/Router.tsx

```diff
@@ -41,7 +41,7 @@ export function Router({ children, withInitializeCommands = false }: RouterProps
         if (childrenArray.some(matchChild('/'))) {
             navigate('/');
         }
-    }, []);
+    }, []); // TODO: childrenArray ?
```

Comment:
Maybe we forgot childrenArray ?

Reply:
Please add disable line

Reply:
please add TODO
// TODO check router children rerendering

Reply:
Fixed